src/notifications/generate.ts

```diff
@@ -552,4 +556,21 @@ export const generateNotificationMap: Record<
       )
       .uniqueKey(`${ctx.post.id}-metrics-${new Date().toISOString()}`);
   },
+  poll_result: (builder: NotificationBuilder, ctx: NotificationPostContext) =>
+    builder
+      .icon(NotificationIcon.Bell)
+      .objectPost(ctx.post, ctx.source, ctx.sharedPost!)
+      .avatarSource(ctx.source)
+      .targetUrl(`${process.env.COMMENTS_PREFIX}/posts/${ctx.post.id}`)
+      .uniqueKey(`${ctx.post.id}-poll-result`),
```

Comment:
you can use `referencePost` which will put `referenceId` to postId and then you don't need custom uniqueKey per this notification type